src/modules/displayItems.js

```diff
@@ -0,0 +1,35 @@
+import displayPopup from './popupWindow.js';
+import { getLikes, likeInteract } from './like.js';
+import itemsCounter from './itemCounter.js';
+
+const recipeCount = document.querySelector('.item-count');
+const recipeCards = document.querySelector('.recipe-cards');
+
+const displayCards = async () => {
+  fetch('https://www.themealdb.com/api/json/v1/1/filter.php?c=Vegetarian')
+    .then((response) => response.json())
+    .then((data) => {
+      let html = '';
+      data.meals.forEach((meal) => {
+        html += `<div class="card" id="${meal.idMeal}">
+               <div class="card-img">
+                   <img src="${meal.strMealThumb}" alt="${meal.strMeal}">
+               </div>
+               <div class="card-info">
+                   <h2>${meal.strMeal}</h2>
+                   <ul>
```

Comment:
✅ When the page loads, the web app should retrieve data from the selected API and show the list of items on the screen. 👍🏻